lib/qbwc/response/all.rb

```diff
@@ -79,10 +79,14 @@ def process(config = {})
           value = value.is_a?(Hash)? [value] : Array(value)
           #value.map(&:values).flatten.select { |value| value.is_a?(Hash) }
 
-          records = value.map { |item| item.values.flatten.select { |value| value.is_a?(Hash) }
-                                           .flatten
-                                           .map { |sub| sub.merge({ 'request_id' => item['@requestID'] }) }
-                              }.flatten
+          records = value.map do |item| 
+            item.values.flatten.map do |value|
+              response_hash = nil
```

Comment:
For some reason it is still showing response_hash here but the branch has already response_item in it, :thinking: not sure what happened.